qa/cephfs/overrides/ignorelist_health.yaml

```diff
@@ -10,4 +10,3 @@ overrides:
       - \(MDS_ALL_DOWN\)
       - \(MDS_UP_LESS_THAN_MAX\)
       - \(FS_INLINE_DATA_DEPRECATED\)
-      - \(POOL_APP_NOT_ENABLED\)
```

Comment:
While I obviously agree with not creating an RBD pool for CephFS tests, I'm surprised to learn that @vshankar's run passed with `POOL_APP_NOT_ENABLED` dropped from the ignorelist.  The issue is not (specific to) the RBD pool, rather it's the raising of a bogus health alert.  See the recent thread on the dev list:

https://lists.ceph.io/hyperkitty/list/dev@ceph.io/thread/ZTDYC5HN677RR26EB4P6PORN6L2IFH4R/

The RBD pool in this case is just the messenger -- it's tagged with the RBD application immediately:
https://github.com/ceph/ceph/blob/636d2a44c7a935426788ed61e9a77432dc49ca5e/qa/tasks/ceph.py#L429-L440

In the example job linked by @batrick in https://tracker.ceph.com/issues/62482, `POOL_APP_NOT_ENABLED` is raised for CephFS pools a few seconds after getting raised on the RBD pool:
```
2023-08-16T11:24:28.204+0000 7fc702096640 20 mgr.server operator() health checks:                    
{                                                                                                        
    "POOL_APP_NOT_ENABLED": {                                                                                  
        "severity": "HEALTH_WARN",                                                                       
        "summary": {                                                                             
            "message": "1 pool(s) do not have an application enabled",                        
            "count": 1                                                                                        
        },                                                                                        
        "detail": [                                                                                
            {                                                                                                 
                "message": "application not enabled on pool 'rbd'"                               
            },                                                                                
            {                                                                                                 
                "message": "use 'ceph osd pool application enable <pool-name> <app-name>', where <app-name> is 'cephfs', 'rbd', 'rgw', or freeform for custom applications."
            }                                                                                                    
        ]                                                                                                     
    }                                                                                                                                                                       
}   
```
```
2023-08-16T11:24:34.204+0000 7fc702096640 20 mgr.server operator() health checks:                    
{                                                                                                        
    "POOL_APP_NOT_ENABLED": {                                                                                  
        "severity": "HEALTH_WARN",                                                                       
        "summary": {                                                                             
            "message": "2 pool(s) do not have an application enabled",                        
            "count": 2                                                                                        
        },                                                                                        
        "detail": [                                                                                
            {                                                                                                 
                "message": "application not enabled on pool 'cephfs_metadata'"                   
            },                                                                                
            {                                                                                                 
                "message": "application not enabled on pool 'cephfs_data'"                                                                                                  
            },                                                                                                   
            {                                                                                                 
                "message": "use 'ceph osd pool application enable <pool-name> <app-name>', where <app-name> is 'cephfs', 'rbd', 'rgw', or freeform for custom applications."
            }                                                                                                      
        ]                                                                                                          
    }                                                                                                                                                                         
}
```

Reply:
I think there was an underlying RADOS change with a fix in-flight for that which maybe was merged when Venky do QA for this. Venky, do you have that link handy still?

Reply:
and ya, this warning came back: https://pulpito.ceph.com/pdonnell-2023-09-05_22:23:53-fs:functional-main-distro-default-smithi/7389194/

Reply:
> I think there was an underlying RADOS change with a fix in-flight for that which maybe was merged when Venky do QA for this. Venky, do you have that link handy still?

Unfortunately, no. I had run a pretty minimal set of test for this and that's probably the reason no failures related to the warning showed up -- just got unlucky I guess?

Reply:
> I had run a pretty minimal set of test for this and that's probably the reason no failures related to the warning showed up -- just got unlucky I guess?

Yeah, this health alert became not only bogus but also racy...